test/test_functions

```diff
@@ -303,6 +305,15 @@ cvmfs_mount() {
   if [ "x$CVMFS_TEST_DEBUGLOG" != "x" ]; then
     sudo sh -c "echo \"CVMFS_DEBUGLOG=$CVMFS_TEST_DEBUGLOG\" >> /etc/cvmfs/default.local" || return 100
   fi
+
+  eval $(cvmfs_config showconfig | grep -e ^CVMFS_CONFIG_REPOSITORY=)
```

Comment:
We should move this into the OS X if block because it is quite a lengthy operation.  Note that `CVMFS_CONFIG_REPOSITORY` is optional.  It might be set but it doesn't need to be.